examples/no-manifest/skaffold.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: skaffold/v1
+kind: Config
+build:
+  # tagPolicy determines how skaffold is going to tag your image
+  # we provide a few strategies here, although you most likely won't need to care!
+  # Valid values for tagPolicy are
+  #
+  # sha256    - tags the image with the checksum of the built image (image id)
+  # gitCommit - tags the image with the git commit of your current repository
+  
+  # tagPolicy: sha256
+
+  # artifacts is a list of the actual images you're going to be building
+  # you can include as many as you want here.
+  artifacts:
+    # The name of the image we're going to build, the full reference to the built
+    # image will be available later in the the deploy section
+  - imageName: changeme # gcr.io/k8s-skaffold/skaffold-example
+    # dockerfilePath: Dockerfile # not required if its just $workspace/Dockerfile
+
+    # workspace is the path to your dockerfile context
+    workspace: examples/getting-started
```

Comment:
examples/no-manifest

Reply:
on second thought, you can leave this like it is and remove the dockerfile and main.go if you want. Then this folder can just be the skaffold.yaml